src/styles.css

```diff
@@ -15,6 +15,10 @@ html {
 	.prose h2 {
 		@apply mt-0;
 	}
+  
+  .heading{
+    color: #F1F5F9 !important;
+  }
```

Comment:
because of the `!important` , I believe this belongs in the `utilities` layer, not `base`.

Would you mind moving it there?

[tailwind layers](https://tailwindcss.com/docs/adding-custom-styles#adding-custom-utilities)